image_tools/test/cam2image.txt

```diff
@@ -0,0 +1,2 @@
+Publishing data on topic 'image'
+Publishing image #1
```

Comment:
Sorry missing a pointer here: based on [this block](https://github.com/ros2/launch/blob/master/launch_testing/launch_testing/__init__.py#L132-L139) I would expect this `cam2image.txt` file to never be taken into account  during the tests